seeding/create-log-records.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#If you're on Windows, run bash create-log-records.sh -w
+#Otherwise, run bash create-log-records.sh
+
+# Import common functions
+source ./seed.sh
+
+# Run SQL script in docker container
+run_sql_script "
+INSERT INTO log_records (
+  employee_id, 
+  resident_first_name, 
+  resident_last_name, 
+  datetime, 
+  flagged, 
+  attn_to, 
+  note, 
+  tags, 
+  building 
+) 
+SELECT
+  $EMPLOYEE_ID,
+  (ARRAY['John','Jerry','James','Jack'])[floor(random() * 4 + 1)], 
+  (ARRAY['Doe','Show','Flow','Mow'])[floor(random() * 4 + 1)], 
+  NOW() + (random() * (NOW()+'90 days' - NOW())) + '30 days', 
```

Comment:
@connor-bechthold this isn't the right date format for the `GET` API, throws this error when called:
```
{
    "error": "'bool' object has no attribute 'astimezone'"
}
```